src/Service/Renderer.php

```diff
@@ -32,8 +32,12 @@ public function __construct(string $rsvgCommand)
     public function render(string $file, string $lang) : string
     {
         $process = new Process([$this->rsvgCommand, $file]);
-        // Set the LANG environment variable, which will be interpreted as the SVG systemLanguage.
-        $process->setEnv(['LANG' => $lang]);
+        if ('fallback' !== $lang) {
+            // Set the LANG environment variable, which will be interpreted as the SVG
+            // systemLanguage. If the fallback language is being requested, the OS's default will be
+            // used instead (as is done in MediaWiki).
+            $process->setEnv(['LANG' => $lang]);
```

Comment:
We want the OS's default, or "en" ? Isn't English the default general fallback for mediawiki languages?

Reply:
We want the SVG's default, which is what this code does.

Reply:
By not setting $LANG, this basically means it'll be en_US, but we don't want to set that within the code because rsvg handles it. And actually, that's a bug with rsvg-convert, because there's no way to specify the fallback if an SVG file contains `en` strings. That's the same everywhere though, so not something svgtranslate needs to worry about I think.